scripts/wait-notify

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
My laptop doesn't recognise 'python', only 'python3'. Should that imply a change on my end (to alias python3, maybe) or a change here?

Reply:
Oh wow I didn't realize Ubuntu/Debian had done that. I'll change it to python3. Thanks!

Reply:
For some context on what I meant - back in the Python 2 days, there was only `python`. Then distributions started shipping both Python 2 and Python 3, as the `python` and `python3` executables respectively (and I think some had `python2` if you wanted to be explicit). Eventually they stopped shipping Python 2 and removed the `python` binary.

Some distributions (off the top of my head at least Arch and NixOS) used that opportunity to now make `python` an alias to `python3`. I did not realise that Debian / Ubuntu never made that alias, and therefore `python` may not exist on these systems.